api/observability/v1/clusterlogforwarder_types.go

```diff
@@ -128,6 +129,13 @@ type CollectorSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Affinity"
 	Affinity *corev1.Affinity `json:"affinity,omitempty"`
+
+	// Define the maxUnavailable pod rollout strategy
+	//
+	// Value can be a number (e.g., 50) or a percentage string (e.g., "50%").
+	// +kubebuilder:validation:Type=string
+	// +kubebuilder:validation:Pattern="^([0-9]{1,2}|100)$|^([0-9]{1,2}|100)%$"
```

Comment:
```suggestion
	// +kubebuilder:validation:Pattern="^(?:[0-9]{1,2}|100)%?$"
```
regexp can be simplified : https://regex101.com/r/LiZGfZ/2